datasets/ollie/README.md

```diff
@@ -0,0 +1,221 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- other-university-of-washington-academic
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+- text-retrieval
+task_ids:
+- entity-linking-retrieval
+- fact-checking-retrieval
+- other-stuctured-to-text

```

Comment:
If I understand correctly the task is relation extraction right ?

```suggestion
task_categories:
- other
task_ids:
- other-other-relation-extraction
```

Reply:
Yes, I'll make the addition  but it's bidrectional though - it's relation extraction and structure to text.  For my purposes, I would also like to experiment with speeding up pre-training using extraction patterns.